projects/optic/src/commands/capture/sources/body.ts

```diff
@@ -25,14 +25,9 @@ export class CapturedBody {
 
   static async json(body) {
     const text = await CapturedBody.text(body);
-    if (text) {
-      return JSON.parse(text);
-    }
+    return JSON.parse(text!);
```

Comment:
Always try to parse this so that we raise an error on undefined or something else (handled in the above case)